examples/latent_sde.py

```diff
@@ -71,23 +71,47 @@ def f(self, t, y):  # Approximate posterior drift.
         inp = torch.cat((torch.sin(t), torch.cos(t), y), dim=-1)
         return self.net(inp)
 
+    def f_aug(self, t, y):
+        y = y[:, 0][:, None]
+        res = self.f(t, y)
+        z = res - self.h(t, y)
+        g = self.g(t, y)
```

Comment:
Why not do something like `f, g, h = self.f(t, y), self.g(t, y), self.h(t, y)`, so that we're clear there's a bunch of function evals?

Reply:
Done!